filter_plugins/openshift_master.py

```diff
@@ -501,7 +501,36 @@ def validate_pcs_cluster(data, masters=None):
                 valid = False
         return valid
 
+    @staticmethod
+    def certificates_to_synchronize(hostvars):
```

Comment:
I decided to base this on hostvars since all of the version booleans have already been created in there.